test/unit/forge/routes/api/project_spec.js

```diff
@@ -1457,4 +1457,272 @@ describe('Project API', function () {
+                outputs: 1,
+                noerr: 0,
+                initialize: '',
+                finalize: '',
+                libs: [],
+                x: 280,
+                y: 120,
+                wires: [
+                    [
+                        'd25b24804605b738'
+                    ]
+                ]
+            },
+            {
+                id: '480ee2eb1cc81556',
+                type: 'http request',
+                z: '7ecba32db86f0617',
+                name: '',
+                method: 'GET',
+                ret: 'txt',
+                paytoqs: 'ignore',
+                url: 'https://jigsaw.w3.org/HTTP/Digest/',
+                tls: '',
+                persist: false,
+                proxy: '',
+                authType: 'digest',
+                senderr: false,
+                x: 330,
+                y: 140,
+                wires: [
+                    [
+                        '0ccbdebc64bd4ff2'
+                    ]
+                ]
+            },
+            {
+                id: '77c84803d42d4f9e',
+                type: 'inject',
+                z: '7ecba32db86f0617',
+                name: '',
+                props: [
+                    {
+                        p: 'payload'
+                    },
+                    {
+                        p: 'topic',
+                        vt: 'str'
+                    }
+                ],
+                repeat: '',
+                crontab: '',
+                once: false,
+                onceDelay: 0.1,
+                topic: '',
+                payload: '',
+                payloadType: 'date',
+                x: 180,
+                y: 140,
+                wires: [
+                    [
+                        '480ee2eb1cc81556'
+                    ]
+                ]
+            },
+            {
+                id: '0ccbdebc64bd4ff2',
+                type: 'debug',
+                z: '7ecba32db86f0617',
+                name: '',
+                active: true,
+                tosidebar: true,
+                console: false,
+                tostatus: false,
+                complete: 'true',
+                targetType: 'full',
+                statusVal: '',
+                statusType: 'auto',
+                x: 470,
+                y: 140,
+                wires: []
+            }
+        ]
+        const credentials = {
+            $: '00fd012ce6da285e6f0fd83e7360afebP/e6OeYPMqTUPXO7L+9e3SUgvJ9NaoCtZbUPme8Y4GMbep8oQzZd64r3J4i1BVvxR5mBZD3kst++ke3114qhm/MmmxyT6V0mogRB0d7z/AapbjnVliXdvE4vGO1tsSHnYa4uf5k//STkK90='
+        }
+        const credsSecret = 'd8bc017ef274d0418725b23d86cdf7a65f8e3699340dd6b65e26600719ad2ac6'
+
+        it('Import Flow', async function () {
```

Comment:
Whilst these tests verify the API returns the expected response, then should also be verifying that the Project's flows have actually been updated.

Reply:
Fair. I'll have a look shortly

Reply:
Test update (and found a bug)